package.json

```diff
@@ -56,11 +56,11 @@
     "copy-webpack-plugin": "~4.4.1",
     "core-object": "^3.1.0",
     "ember-cli-string-utils": "^1.0.0",
-    "extract-text-webpack-plugin": "^3.0.2",
+    "extract-text-webpack-plugin": "~4.0.0-alpha.0",
     "file-loader": "^1.1.5",
     "fs-extra": "^4.0.0",
     "glob": "^7.0.3",
-    "html-webpack-plugin": "^2.29.0",
+    "html-webpack-plugin": "github:webpack-contrib/html-webpack-plugin#a8a8c2b6ea496c257fd6f501db3a06a51fa03e1e",
```

Comment:
The PR says "CLI internal index html plugin used" https://github.com/angular/angular-cli/pull/9459#issue-166323121.
Why do we need the html-webpack-plugin then? Is it only a temporary fix?

Reply:
See text below headline „🛑 If you use HtmlWebpackPlugin“ on https://medium.com/webpack/webpack-4-released-today-6cdb994702d4

Reply:
Yes, I know this. But @clydin wrote "CLI internal index html plugin used" instead, that is why I am asking.

Reply:
Looks like the article got updated a couple days after this question:
> UPDATE (3/1/2018): html-webpack-plugin@3 is now available with v4 support!!!!